### PR TITLE
Server SPI Package Re-Organising

### PR DESCRIPTION
**Pull request by Jarota**
I've updated the package statements but I can't manage to figure out how to move the spi folder into event folder. 😞

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/665)

### CODE REVIEW

**Comment by hendrikebbers** [comment]
Normally you can simply move the folder. What is your problem? Maybe we can help :)

**Comment by Jarota** [comment]
I was thinking that'd make sense! If I click and drag spi over event there's an X on the tooltip and nothing happens..

**Comment by hendrikebbers** [comment]
What IDE are you using?

**Comment by Jarota** [comment]
Oh I was trying to move it around just on GitHub, oops

**Comment by Jarota** [comment]
I think I've got it (?)

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<projectDescription>
+	<name>dolphin-platform</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+	</natures>
+</projectDescription>
```
Comment (hendrikebbers) [inline .project]: Please remove this file or add it to .gitignore

**Comment by hendrikebbers** [comment]
Hi @Jarota you moved the package to the right place. But Dolphin Platform contains implementations of the interface and SPI related files in META-INF/services of several modules that are related to the interface. Since all this files are not changed Dolphin Platform will not compile anymore based on your change. When doing such a refactoring with an IDE like IntelliJ you would see it or the IDE would automatically refactor the related files based on your change.

**Pull request closed without merging**